app/javascript/styles/components.scss

```diff
@@ -2653,19 +2653,36 @@ button.icon-button.active i.fa-retweet {
   flex-direction: column;
 }
 
-@keyframes pulse {
-  0% {
-    opacity: 1;
+@keyframes heartbeat {
+  from {
+    transform: scale(1);
+    transform-origin: center center;
```

Comment:
`transform-origin: center center;` should normally go inside of the `.pulse-loading {}` instead of here; this might technically work in some browsers but seems safer to me not to put it here.